web/app/controllers/dashboard.js

```diff
@@ -18,13 +18,21 @@ import Ember from 'ember';
 
 export default Ember.Controller.extend({
   showInputBox: false,
+  notifications: Ember.inject.service('notification-messages'),
   actions: {
 
     /**
      * This action adds a new tab and clicks on it once the tab is added and rendered
      * @params user The user to be added as a tab
      */
     addTab(user) {
+
+      if(user==null || user=="") {
```

Comment:
Please use ===. See this [stack overflow article](http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons) for reference